django/core/jinja_config.py

```diff
@@ -182,6 +185,14 @@ def format_datetime(date_obj, format_string=FULL_DATE_FORMAT):
     return date_obj.strftime(format_string)
 
 
+def timeuntil(date):
+    if date is None:
+        return None
+    elif isinstance(date, str):
+        date = parse_datetime(date)
+    return timesince(date, timezone.now())
```

Comment:
Sure is, no clue what I was thinking here since this is just wrapping `timesince` for str input (and being used for time since)  and calling it `timeuntil`

Showing time until as well is a good idea, I'm thinking that could go where the expired badge is currently